### PR TITLE
issue #125

### PR DESCRIPTION
**Pull request by pvilledieu**
Hi Florian, 
Let me know if it's close from what you expect. Then I'll do the final adjustments.

### CODE REVIEW

File: 09-projects.Rmd [hunk 1]
```diff
@@ -24,45 +24,243 @@ In order to achieve those goals, you will interact with the Atlas website at [ht
...
+# Tract 24003706500, Annapolis, MD
+```
+    1. Create Figure 1 in your report, which should display the map shown by [https://www.opportunityatlas.org](https://www.opportunityatlas.org) for the census tracts around your home (you can download that map *as an image* bottom left).
+```{r out.width = "75%", echo = FALSE, eval = FALSE}
+library(knitr)
+include_graphics("images/pv-atlas.png")
+```
+    1. Your accompanying text should describe what is shown in that map, in particular what data are being visualized. Examine next the patterns for a number of different groups (e.g., lowest income children, high income children) and outcomes (e.g., earnings in adulthood, incarceration rates).  Only choose one or two of these to include in your report.
+```{r out.width = "75%", echo = FALSE, eval = FALSE}
+library(knitr)
```
Comment (floswald) [inline 09-projects.Rmd]: load `knitr` only once

File: 09-projects.Rmd [hunk 2]
```diff
@@ -24,45 +24,243 @@ In order to achieve those goals, you will interact with the Atlas website at [ht
...
+  
     ```R
     # explain lm
     # explain scatter plot
     ```
+        
+```{r, echo = FALSE, eval = FALSE}
+library(dplyr)
+library(ggplot2)
+c = readRDS(system.file(package = "ScPoEconometrics","datasets","atlas.Rds")) %>%
```
Comment (floswald) [inline 09-projects.Rmd]: data still in memory as `d`? no need to reload.

File: 09-projects.Rmd [hunk 3]
```diff
@@ -24,45 +24,243 @@ In order to achieve those goals, you will interact with the Atlas website at [ht
...
+```
+    
 1. Next, examine whether the patterns you have looked at above are similar by race. If there is not enough racial heterogeneity in the area of interest (i.e., data is missing for most racial groups), then choose a different area to examine.
+
+```{r, echo = FALSE, eval = FALSE}
+library(dplyr)
+library(ggplot2)
+
+# Home county 
+c = readRDS(system.file(package = "ScPoEconometrics","datasets","atlas.Rds")) %>%
```
Comment (floswald) [inline 09-projects.Rmd]: same data loading. maybe it's good to leave comments in there to that effect!

File: _slides/chapter1/chapter1.md [hunk 4]
```diff
@@ -5,7 +5,7 @@
 ScPoEconometrics
 ========================================================
 author: Florian Oswald
-date: 2018-09-14
+date: 2019-07-29
```
Comment (floswald) [inline _slides/chapter1/chapter1.md]: sorry i didnt tell you but I'm retiring this section with `_slides`. slides will be hosted in a separate repo ;-)

**Comment by floswald** [comment]
ok, so it's easy. we just set in an initial code chuck that

````
```{r setup, echo=FALSE}
showsols <- TRUE
```
````

text

````
```{r sol1, echo = showsols}
# solution
```
````

File: _slides/chapter1/chapter1.md (hunk 4 above)
Comment (pvilledieu) [reply _slides/chapter1/chapter1.md]: Do I need to pull again from your repo to have the exact same version (except the file I'm editing) ?

Comment (floswald) [reply _slides/chapter1/chapter1.md]: don't worry about that part - just leave as is. your version is mergeable with the master branch. i'm removing the slides on a different branch.
you can just push onto that same branch that you created this PR from.

**Comment by floswald** [comment]
thanks carried on in #127

**Pull request closed without merging**